subjects/skiptheline/README.md

```diff
@@ -0,0 +1,38 @@
+## AddFront
+
+### Instructions
+
+Write a function that receives a string and a slice of strings. Return a new slice with the given string prepended.
+
+### Expected function 
+
+```go
+func AddFront(s string, slice []string) []string {
+    // your code here
+}
+```
+
+### Usage
+
+Here is a possible program to test your function:
+
+```go
+package main
+
+import "fmt"
+
+func main() {
+    fmt.Println(AddFront("Hello", []string{"world"}))
+    fmt.Println(AddFront("Hello", []string{"","world", "!"}))
+    fmt.Println(AddFront("Hello", []string{}))
+}
+```
+
+and the output should be:
+
+```console
+$ go run .
+[Hello world]
+[Hello world !]
```

Comment:
When I run this program for this example i get this:
```[Hello  world !]``` with an extra space in the middle, but it is difficult to tell by looking at it, wouldn't it be better to compare the length of the slices instead of printing them?